tests/embed/test_splitae.py

```diff
@@ -0,0 +1,17 @@
+from multiview.embed.splitae import SplitAE
+import numpy as np
+
+def test_splitae_overfit():
```

Comment:
this is not enough testing at all. need input validation at a bare minimum. I also dont understand what looking at the standard deviation here is telling you about the success of the method

Reply:
so you also need some kind of better test that your thing is generating a good embedding, not just that it is overfitting...

Reply:
stdev check is of (actual-predicted) to assure overfitting while allowing for some outliers (e.g. `np.all(np.abs(actual-predicted) < threshold)` would fail sometimes)

Reply:
Overfit check is usually the first thing that's done when making sure a network is working but maybe I can do something where I set the torch seed and look for certain results

Reply:
my point is, why does this metric show us that the method is working? I don't think overfitting is a good test

Reply:
* i should say, it isn't sufficient as the only test

Reply:
I agree that it's not sufficient, I think a better test is looking at the Noisy MNIST embeddings as in the notebook, maybe I can squeeze that into this test suite somehow.

Reply:
FTR, our other testing suites (at least. [test_omnibus](https://github.com/NeuroDataDesign/multiview/blob/8000bd061c8506daa8d24662ac44ec6dcb85f8b9/tests/test_omnibus.py )) are not testing that a good embedding is found. (e.g.  our Omnibus embedding tests are only checking that the number of views input = the number of views output). Despite it seeming like there is a lot of testing there.